tests/cephfs/cephfs_scale/run_scale_io.py

```diff
@@ -92,6 +92,26 @@ def run(ceph_cluster, **kw):
                 )
                 log.info(f"Ceph fs status after enabling standby-replay : {result}")
 
+        # nfs_servers = ceph_cluster.get_ceph_objects("nfs")
+        # nfs_server = nfs_servers[0].node.hostname
+        nfs_server = "extensa023"
```

Comment:
This is hardcoded for the time being.. 
As we will be using a dedicated Scale Cluster.